### PR TITLE
build Olena without Qt parts

### LINKED ISSUE #29: does not compile (within ocrd_all, Ubuntu 18.04)
```
jb@pers16:~/ocrd1804/ocrd_all> git log|head 
commit 0f7d5023992d19faa497096d445d88571fa1dbfe
Merge: 84bc669 f35fd3b
Author: Stefan Weil <sw@weilnetz.de>
Date:   Tue Feb 11 15:56:51 2020 +0100

    Merge pull request #61 from OCR-D/update-2020-02-11
    
    update 2020-02-11
```

```
jb@pers16:~/ocrd1804/ocrd_all/ocrd_olena> git log|head 
commit e0d8858fa24ca63bb1047045cc9ed05fdbb256fb
Merge: 55e8042 6e76bfe
Author: Konstantin Baierer <kba@users.noreply.github.com>
Date:   Thu Feb 6 14:56:43 2020 +0100

    Merge pull request #27 from stweil/work
    
    Add new macro BUILD_DIR for build directory

commit 6e76bfe3123996f4df0f13c3f2db1067dda280b0
```

`make all` within ~/ocrd1804/ocrd_all raises this error:

```
make[6]: Verzeichnis „/home/jb/ocrd1804/ocrd_all/venv/build/ocrd_olena/scribo/demo/xml2doc“ wird betreten
/usr/bin/moc -DQT_DEPRECATED_WARNINGS -DQT_NO_DEBUG -DQT_GUI_LIB -DQT_XML_LIB -DQT_CORE_LIB -I. -I. -isystem /usr/include/x86_64-linux-gnu/qt5 -isystem /usr/include/x86_64-linux-gnu/qt5/QtGui -isystem /usr/include/x86_64-linux-gnu/qt5/QtXml -isystem /usr/include/x86_64-linux-gnu/qt5/QtCore -I. -isystem /usr/include/libdrm -I/usr/lib/x86_64-linux-gnu/qt5/mkspecs/linux-g++ /home/jb/ocrd1804/ocrd_all/ocrd_olena/repo/olena/scribo/demo/xml2doc/common.hh -o common.moc.cc
Unknown options: isystem, isystem, isystem, isystem, isystem.
```

### PR DESCRIPTION
**Pull request by bertsky**
Fix #29

### CODE REVIEW

**Review by stweil — APPROVED** [review]

**Comment by stweil** [comment]
I'd update OCR-D/ocrd_all directly with this fix as soon as it is merged, ok?

**Comment by bertsky** [comment]
> I'd update OCR-D/ocrd_all directly with this fix as soon as it is merged, ok?

Yes, it's perhaps not necessary to wait for confirmation from @jbarth-ubhd. This is a trivial fix.

I'll merge here as soon as CI gives green light.

**Comment by stweil** [comment]
A nice side effect of this patch is a faster build with less disk storage needed. Maybe even more parts can be disabled because they are not required for OCR-D?

**Pull request merged**
